cluster/gce/config-default.sh

```diff
@@ -245,12 +245,16 @@ if [ ${ENABLE_IP_ALIASES} = true ]; then
   PROVIDER_VARS="${PROVIDER_VARS:-} ENABLE_IP_ALIASES"
 fi
 
-
 # Enable GCE Alpha features.
 if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
   PROVIDER_VARS="${PROVIDER_VARS:-} GCE_ALPHA_FEATURES"
 fi
 
+# Disable Docker live-restore.
+if [[ -n "${DISABLE_DOCKER_LIVE_RESTORE:-}" ]]; then
```

Comment:
How about checking whether it's `true`/`false` so that `DISABLE_DOCKER_LIVE_RESTORE=false` will work as expected?

Reply:
Done. See the latest commit.